### PR TITLE
Remove several uses of deprecated `MindmapsGraph#put` methods

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

**Comment by alexandraorth** [comment]
[This change makes Bulbasaur sad.
](http://img09.deviantart.net/034b/i/2010/194/a/a/sad_bulbasaur_by_thenekohimechan.jpg)
```
  Entity bulbasaur = mindmapsGraph.getEntity("Bulbasaur");
```

to

```
Entity bulbasaur = mindmapsGraph.getResourcesByValue("Bulbasaur").iterator().next().ownerInstances().iterator().next().asEntity();
```

I don't think we should be making these changes until we have a better way to query by resource -- we will just have to do this all again when we do get that method.

**Comment by aelred** [comment]
I agree with @alexandraorth. The above code is only the tip of the iceberg for ugly queries! Once we have the unique resource method, it will be a little better, but still not pretty:

```
ResourceType<String> name = mindmapsGraph.getResourceType(name, STRING);
Entity bulbasaur = mindmapsGraph.getResource("Bulbasaur", name).ownerInstances().iterator().next().asEntity();
```

**Comment by aelred** [comment]
please retest this please

File: mindmaps-graph/src/main/java/io/mindmaps/example/PokemonGraphFactory.java [hunk 1]
```diff
@@ -47,6 +48,26 @@
     private static RelationType evolution;
     private static RelationType hasType;
     private static RelationType superEffective;
+    private static Entity normal;
```
Comment (fppt) [inline mindmaps-graph/src/main/java/io/mindmaps/example/PokemonGraphFactory.java]: Hahahaha I remember doing this when we first took a whack at internalising ids :)

**Comment by fppt** [comment]
I will work on creating the unique resources feature. It will make things better. We just need to focus on getting rid of all these puts so we can find the issues caused by this change faster.

**Pull request merged**